tokens/utils.js

```diff
@@ -162,7 +162,37 @@ async function transformInPath(location, variablesMap, transformType = 'definiti
   }
 }
 
+
+function createIndexFile(buildDir, isTheme, themeVariant){
+  const directoryPath = isTheme ? `${buildDir}/themes/${themeVariant}`:`${buildDir}/core`;
+  
+fs.readdir(directoryPath, (err, files) => {
+  if (err) {
+    console.error('Error reading directory:', err);
+    return;
+  }
+
+  const jsonFiles = files.filter(file => file !== 'index.css');
```

Comment:
[clarification] Is this variable named `jsonFiles` because the resulting file names are the output from building the JSON design tokens? If so, this wasn't immediately obvious where for awhile I was thinking the `files` contains file names with `.json` file extensions but I don't think that's the case? If we're solely working with CSS files here, I wonder if there's a clearer name here, e.g. `nonIndexCssFiles` or `outputCssFiles`, etc.